doc/source/release-notes/Percona-Server-8.0.16-7.rst

```diff
@@ -0,0 +1,61 @@
+enterprise-grade features developed by Percona.  For a list of highlighted
+@@ -35,7 +35,7 @@ features from both MySQL 8.0 and Percona Server for MySQL 8.0, please see the
+New Features
+================================================================================
+
+- `Key rotation for the redo log <https://www.percona.com/doc/percona-server/LATEST/management/data_at_rest_encryption.html>`__ has been redesigned to rotate the key with `SELECT rotate_system_key("percona_redo"). 
+
+Bugs Fixed
+================================================================================
+- The :variable: `buf_dblwr_flush_buffered_writes` must crash the server when an I/O error occurs to the parallel doublewrite buffer. The doublewrite buffer error returns to the caller. Bug fixed :psbug:`5678`.
+- After resetting the :variable:`innodb-temp-tablespace-encrypt` to `OFF` during runtime the subsequent file-per-table temporary tables continue to be encrypted. Bug fixed :psbug:`5734`.
+- The `key rotation process <https://www.percona.com/doc/percona-server/LATEST/management/data_at_rest_encryption.html>`__ was redesigned. The updated process allows the key rotation with `SELECT rotate_system_key("percona_redo")` and the currently used key version is displayed in the `innodb_redo_key_version` variable.  Bug fixed :psbug:`5565`.
+- Setting the encryption `ON` for the system tablespace generates an encryption key and encrypts system temporary tablespace pages. Resetting the encryption to `OFF`, all subsequent pages are written to the temporary tablespace without encryption. To allow any encrypted tables to be decrypted, the generated keys are not erased. Modifying the :variable:`innodb_temp_tablespace_encrypt` does not affect file-per-table temporary tables. This type of table is encrypted if `ENCRYPTION='y'` at table creation.  Bug fixed :psbug:`5736`.
+- An instance started with the default values but setting the redo log to encrypt without specifying the keyring plugin parameters does not fail or throw an error.  Bug fixed :psbug:`5476`.
+- The :variable: `rocksdb_large_prefix` allows index key prefixes up to 3072 bytes. The default value is changed to `TRUE` to match the behavior of the :variable:`innodb_large_prefix`. :psbug:`5655 `.
+- On a server with two million or more tables, a user experiences a shutdown in less than ten minutes.  Bug fixed :psbug:`5639`.
+- The changed page tracking may read pages which have the encryption bit set.  The redo log does not have the encryption key and decryption fails and generates an error message. A `flag <https://www.percona.com/doc/percona-server/LATEST/management/data_at_rest_encryption.html>`__ lets the read process safely ignore encryption errors in this case.  Bug fixed :psbug:`5541`.
+- Large page allocations with the :variable:`innodb_buffer_pool_chunk_size` set to a shared memory segment larger than 4GB or more generates an incorrect size. Bug fixed :psbug:`5517`.
```

Comment:
Please be more specific about shared memory segment, I don't think InnoDB buffer pool uses those specifically

Reply:
rewritten: If large pages are enabled on MySQL side, the maximum size for :variable:`innodb_buffer_pool_chunk_size` is effectively limited to 4GB.